### ISSUE TITLE
Setup your AWS sandbox account

### ISSUE THREAD

**Issue by psirenny** [spear-ai/hackthemachine-unmanned-track-3#6]
@spear-ai/salacious-crumb

Everyone on the team has a sandbox AWS account to play with. You may have received a strange email from AWS over the weekend when your HACKtheMACHINE account was created.

Here's a list of each root account:
* Alex Nickel: alnickel+htmu-user@amazon.com
* Amy Tippl: attippl+htmu-user@amazon.com
* Andrew Maksimovich: andrewma+htmu-user@amazon.com
* Chris Canniff: ccanniff1+htmu-user@gmail.com
* Dennis Torres: dtorres+htmu-user@spear.ai
* Jaim Coddington: jcoddington+htmu-user@spear.ai
* ~Jackson Koehler: jackson.koehler+htmu-user@usacet.com~
* Jackson Koehler: tjacksonkoehler+htmu-user@gmail.com
* Joseph Obeid: jobeid+htmu-user2@spear.ai
* Kevin Rosa: kevin+htmu-user@current-lab.com
* Michael Hunter: mhunter+htmu-user@spear.ai

**Note:** If there's an issue with any of the email addresses then please let me know!

Use your sandbox account to provision AWS resources that you'll need for the hackathon. And if you spin up a resource that you no longer need then please remember to terminate it 😄 

**How do I access my account?**

1. Visit [https://signin.aws.amazon.com](https://signin.aws.amazon.com/signin?redirect_uri=https%3A%2F%2Fconsole.aws.amazon.com%2Fconsole%2Fhome%3Ffromtb%3Dtrue%26hashArgs%3D%2523%26isauthcode%3Dtrue%26nc2%3Dh_ct%26src%3Dheader-signin%26state%3DhashArgsFromTB_us-east-1_cd7296b31a5edeea&client_id=arn%3Aaws%3Asignin%3A%3A%3Aconsole%2Fcanvas&forceMobileApp=0&code_challenge=fc9XXIokVWMCUrK27WAqce_o_9Vbl-FCgmCk3V00YNY&code_challenge_method=SHA-256)
2. Click sign in as **Root User**
3. Enter your email address
4. Enter the Captcha information
5. Click __Forgot password?__
6. Follow the password reset instructions (you'll need access to the account email address)
7. Reset your password
8. Sign in with your new password

**Additional steps**

I recommend creating an IAM account for yourself in the console and signing in that way. Also, if you want to use the AWS CLI or CDK then you'll _need_ an IAM account:

1. Go to IAM in the console
2. Create a new user
3. Assign the user the **AdministratorAccess** permission
4. Optionally Setup MFA (multi-factor authentication)
5. Generate access key credentials

Save your credentials to the `.env` file at the root of your project. Example:

```sh
AWS_ACCESS_KEY_ID=…
AWS_SECRET_ACCESS_KEY=…
CDK_DEFAULT_ENV_NAME=user-…
```

The `.env` file isn't checked into version control because it contains sensitive information (your credentials).

**Comment by psirenny** [comment]
@captainjackcity you should use the account to setup the beanstalk app. You'll be the first person (other than me) to run through those steps. Please let me know what issue --if any -- you run into along the way.

**Comment by psirenny** [comment]
BTW here's a useful list of each account which can come in handy if you need to quickly lookup your account id:
https://github.com/spear-ai/hackthemachine-unmanned-track-3/blob/main/cdk.json#L23

**Comment by psirenny** [comment]
@spear-ai/salacious-crumb I made an assumption that everyone's email service supported sub-addressing (plus addressing). That assumption was false. If you go to reset your password but _don't_ receive a password reset email from AWS, then let me know. We'll figure out another email address to use that doesn't have a plus symbol in it.

BTW, I used sub-addressing to avoid reserving an AWS account under your primary email address. These are root AWS accounts, so your email address uniquely identifies who you are. I thought it would be obnoxious to automatically reserve your email address just for the purposes of this hackathon. However, if you don't mind... then we can use that too.
